contracts/votingMachines/GenesisProtocolLogic.sol

```diff
@@ -140,7 +140,7 @@ contract GenesisProtocolLogic is IntVoteInterface {
       // solium-disable-next-line security/no-block-members
         require(now > parameters[_paramsHash].activationTime,"not active yet");
         //Check parameters existence.
-        require(parameters[_paramsHash].queuedVoteRequiredPercentage > 0);
+        require(parameters[_paramsHash].queuedVoteRequiredPercentage >= 50);
```

Comment:
don't understand why do this here when it was already done in setParameters?

Reply:
please read my previous comment on that . it is used here to check the params are valid (not zero)

Reply:
I'm sorry I'm finding another comment on this.  I don't understand how the param value could have changed between `setParameters` and `propose`

Reply:
_paramsHash is parameter to the propose function.